core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -277,72 +277,50 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
           <em>There are no collections to display.</em>
         </div>
 
-        <ul class="oppia-dashboard-tiles" ng-if="activeTab === 'myCollections'">
-          <li ng-repeat="collection in collectionsList">
-            <md-card layout="row" class="oppia-dashboard-tile">
-              <a ng-href="/collection_editor/create/<[collection.id]>" class="oppia-dashboard-tile-container-link" style="text-decoration: none;">
-                <div class="oppia-dashboard-tile-metadata-parent hidden-xs">
-                  <div class="oppia-dashboard-tile-metadata">
-                    Created:<span style="color: black;">
-                      <[getLocaleAbbreviatedDatetimeString(collection.created_on)]>
-                    </span>
-
-                    <br>
-                    <br>
-                    <br>
-
-                    Status:
-                    <span ng-if="collection.status !== 'private'">
-                      <span class="oppia-dashboard-status-green">
-                        Published
-                      </span>
-                    </span>
-                    <span ng-if="collection.status === 'private'">
-                      <span class="oppia-dashboard-status-grey">
-                        Private
-                      </span>
-                    </span>
+        <ul class="oppia-dashboard-tiles"
+            ng-if="activeTab === 'myCollections' && collectionsList.length > 0">
+          <md-card class="oppia-activity-summary-tile oppia-dashboard-collection-tile bottom-card"
```

Comment:
Not possible since clicking on the tile made out of 'collection summary tile directive' leads to playing the collection, and not opening the collection editor. I had thought of adding a flag for this, but instead used a modified version of that directive's code here so that it's easy to keep things seperate (b/w library collection tiles & dashboard collection tiles) without having to add too many of flags to the directive!